### ISSUE TITLE
Augmentation during linear evaluation

### ISSUE THREAD

**Issue by chenwydj** [leftthomas/SimCLR#3]
It seems the SimCLR paper did not use augmentation during linear evaluation. Shall we use the `test_transform ` here? 
https://github.com/leftthomas/SimCLR/blob/master/linear.py#L72

**Comment by leftthomas** [comment]
@chenwydj you could look what really does `test_transform`  do, it just normalize the data, no any other data augmentation, every model will do this.

**Comment by chenwydj** [comment]
Hi @leftthomas ! Thank you for your response!

What I mean is, you are actually using `train_transform` during the linear evaluation training.
https://github.com/leftthomas/SimCLR/blob/master/linear.py#L72

While in page 14 section "Linear Evaluation" in SimCLR paper, they said "We used L-BFGS to optimize the softmax cross-entropy objective and we did not apply data augmentation."

May I know if there is any misunderstanding here?

Thanks!
